cmd/uplink/cmd/setup_test.go

```diff
@@ -115,3 +120,59 @@ func TestDefaultPortAppliedToSatelliteAddrWithPortColonButNoPort(t *testing.T) {
 	assert.Equal(t, "ahost:7777", setupCmd.Flags().Lookup("satellite-addr").Value.String(),
 		"satellite-addr should contain default port when no port specified")
 }
+
+func TestSaveEncriptionKey(t *testing.T) {
+	expKey := make([]byte, rand.Intn(20)+1)
+	_, err := rand.Read(expKey)
+	require.NoError(t, err)
+
+	t.Run("ok", func(t *testing.T) {
+		dir, err := ioutil.TempDir("", "storj-test-cmd-uplink")
```

Comment:
I wasn't aware of such package for doing such things, I will use it instead of.
Taking a look what it has, there should be more places to use it, so I'll check them too.